pkgsrc/switch-p4-16/api/common/pal.cpp

```diff
@@ -1803,14 +1803,14 @@ switch_status_t speed_set(switch_object_id_t port_handle,
   port_lane_list.v_set(lane_list);
   status = switch_store::attribute_set(port_handle, port_lane_list);
   if (status != SWITCH_STATUS_SUCCESS) {
-    switch_log(SWITCH_API_LEVEL_ERROR,
-               SWITCH_OBJECT_TYPE_PORT,
-               "{}.{}: lane_list failure port_handle {} speed {} status {}",
-               __func__,
-               __LINE__,
-               port_handle,
-               port_lane_list,
-               status);
+    // switch_log(SWITCH_API_LEVEL_ERROR,
+    //            SWITCH_OBJECT_TYPE_PORT,
+    //            "{}.{}: lane_list failure port_handle {} speed {} status {}",
+    //            __func__,
+    //            __LINE__,
+    //            port_handle,
+    //            port_lane_list,
+    //            status);
```

Comment:
Do you want to merge this PR with this code commented out?  Or perhaps wrap them all in an `#ifdef SOME_NEW_SYMBOL_NAME` with the same symbol name as each other, to make them easier to grep and find later when someone wants to update all of these calls?  I guess marking them all with an identical unique string in a comment works for that, too.

Reply:
I will remove these changes later. But I need them to build on Ubuntu 24.04, which is the distribution I am currently using. 

The problem is that this code doesn't compile with newer version of GCC/CLang, which enforce the C++ standard correctly.